### ISSUE TITLE
VS Code: crash on skywalk's computer

### ISSUE THREAD

**Issue by strager** [quick-lint/quick-lint-js#455]
```
D:\a\quick-lint-js\quick-lint-js\src\change-detecting-filesystem-win32.cpp:172: internal check failed in watch_directory: false
```

version 0.4.0

**Comment by strager** [comment]
This might be half-fixed already. I should deploy a new version so we can get an error pop-up instead of a crash.

**Comment by strager** [comment]
skywalk was using a network share. A newer version fixes the crash.

**Comment by strager** [comment]
Shipped 0.5.0 with a partial fix. (We fixed the crash, but service is degraded for network shares.) https://quick-lint-js.com/releases/
